From 2a0edcd6b7666511ee64e0cd4cfb667980f5ae57 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sat, 9 May 2020 13:18:50 -0700
Subject: [PATCH 3/3] Add .3 to Release string

---
 glibc.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glibc.spec b/glibc.spec
index ba6dd48..d46c55e 100644
--- a/glibc.spec
+++ b/glibc.spec
@@ -103,7 +103,7 @@ Patch400002: 0001-x86-Don-t-hardcode-thresholds-for-rep-movsb-stosb.patch
 Summary: The GNU libc libraries
 Name: glibc
 Version: %{glibcversion}
-Release: 2.2%{?dist}
+Release: 2.3%{?dist}
 
 # In general, GPLv2+ is used by programs, LGPLv2+ is used for
 # libraries.
-- 
2.26.2

