From fbe0b9516bf66f50f44a46d35b5c3e8400397635 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Thu, 19 Mar 2020 07:57:06 -0700
Subject: [PATCH 7/7] Define buildid to .1.cet

---
 kernel.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel.spec b/kernel.spec
index 96f61842c..8559ff7a3 100644
--- a/kernel.spec
+++ b/kernel.spec
@@ -102,6 +102,7 @@ Summary: The Linux kernel
 %endif
 
 # define buildid .local
+%define buildid .1.cet
 
 %if 0%{?fedora}
 %define primary_target fedora
-- 
2.26.2

