From 39a5faf990651de3941f87183404b2ed9f5b1da6 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sun, 12 Apr 2020 11:20:43 -0700
Subject: [PATCH 2/2] Add .1 to Release string

---
 gcc.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc.spec b/gcc.spec
index 5fcc396..b51ff7d 100644
--- a/gcc.spec
+++ b/gcc.spec
@@ -122,7 +122,7 @@ Patch100001: 0001-x86-Restore-the-frame-pointer-in-word_mode.patch
 Summary: Various compilers (C, C++, Objective-C, ...)
 Name: gcc
 Version: %{gcc_version}
-Release: %{gcc_release}.0%{?dist}
+Release: %{gcc_release}.1%{?dist}
 # libgcc, libgfortran, libgomp, libstdc++ and crtstuff have
 # GCC Runtime Exception.
 License: GPLv3+ and GPLv3+ with exceptions and GPLv2+ with exceptions and LGPLv2+ and BSD
-- 
2.26.0

