From d9929c4f6d5829139ab94cf8a787407f7520a2ab Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sun, 17 May 2020 11:12:33 -0700
Subject: [PATCH 2/2] Add .1 to Release string

---
 gcc.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc.spec b/gcc.spec
index 1f88453..affe669 100644
--- a/gcc.spec
+++ b/gcc.spec
@@ -123,7 +123,7 @@ Patch100002: 0002-x86-Allow-V1TI-vector-register-pushes.patch
 Summary: Various compilers (C, C++, Objective-C, ...)
 Name: gcc
 Version: %{gcc_version}
-Release: %{gcc_release}.0%{?dist}
+Release: %{gcc_release}.1%{?dist}
 # libgcc, libgfortran, libgomp, libstdc++ and crtstuff have
 # GCC Runtime Exception.
 License: GPLv3+ and GPLv3+ with exceptions and GPLv2+ with exceptions and LGPLv2+ and BSD
-- 
2.26.2

