From fa8770201eb085ff8da88b403d8862cb1d2cae92 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Tue, 7 Apr 2020 06:16:51 -0700
Subject: [PATCH] Updated to 20200407

---
 binutils.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/binutils.spec b/binutils.spec
index b1d411c..4e95fbf 100644
--- a/binutils.spec
+++ b/binutils.spec
@@ -105,8 +105,8 @@ URL: https://sourceware.org/binutils
 # too many controversial patches so we stick with the official FSF version
 # instead.
 
-%global DATE 20200403
-Epoch: 255
+%global DATE 20200407
+Epoch: 256
 Source: binutils-%{version}-%{DATE}.tar.bz2
 #Source: https://ftp.gnu.org/gnu/binutils/binutils-%{version}.tar.xz
 
-- 
2.26.0

