From 5dfc134b24d91e2512cdca00660c1a0879a84635 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Fri, 21 Feb 2020 10:02:32 -0800
Subject: [PATCH 2/2] Add .0 to Release string

---
 mozjs60.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mozjs60.spec b/mozjs60.spec
index 6579b83..90cfe19 100644
--- a/mozjs60.spec
+++ b/mozjs60.spec
@@ -20,7 +20,7 @@ Patch100003: 0003-Insert-ENDBR64-at-entries-of-jitted-code.patch
 
 Name:           mozjs%{major}
 Version:        60.9.0
-Release:        5%{?dist}
+Release:        5.0%{?dist}
 Summary:        SpiderMonkey JavaScript library
 
 License:        MPLv2.0 and MPLv1.1 and BSD and GPLv2+ and GPLv3+ and LGPLv2+ and AFL and ASL 2.0
-- 
2.26.0

