From d0e347e57f8d1106161f4b195616a58f3ca4a9c0 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Wed, 29 Apr 2020 15:44:33 -0700
Subject: [PATCH] Define buildid to .1.cet

---
 kernel.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel.spec b/kernel.spec
index 764758825..912c43508 100644
--- a/kernel.spec
+++ b/kernel.spec
@@ -109,7 +109,7 @@ Summary: The Linux kernel
 %global zipsed -e 's/\.ko$/\.ko.xz/'
 %endif
 
-%define buildid .5.cet
+%define buildid .1.cet
 
 
 %if 0%{?fedora}
-- 
2.26.2

