From eb2ab661fbd6c8d1dc598c61e4a7c4b17239523a Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Thu, 19 Mar 2020 07:57:06 -0700
Subject: [PATCH 5/8] Define buildid to .1.cet

---
 kernel.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel.spec b/kernel.spec
index c4d210a8e..5d1aa8ec3 100644
--- a/kernel.spec
+++ b/kernel.spec
@@ -105,6 +105,7 @@ Summary: The Linux kernel
 %endif
 
 # define buildid .local
+%define buildid .1.cet
 
 %if 0%{?fedora}
 %define primary_target fedora
-- 
2.26.2

