From b03c813d10f74107777295dbaad6473d853e28fc Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Thu, 19 Mar 2020 07:57:06 -0700
Subject: [PATCH 5/5] Define buildid to .1.cet

---
 kernel.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel.spec b/kernel.spec
index 66a96e11e..538cbc547 100644
--- a/kernel.spec
+++ b/kernel.spec
@@ -110,6 +110,7 @@ Summary: The Linux kernel
 %endif
 
 # define buildid .local
+%define buildid .1.cet
 
 
 %if 0%{?fedora}
-- 
2.26.0

