From 270a3b64b0d9deab8d5ba57f149febf20dec138a Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Mon, 24 Feb 2020 03:23:27 -0800
Subject: [PATCH 1/2] Add .0 to Release string

---
 python3.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python3.spec b/python3.spec
index 05c6360..2b54bd3 100644
--- a/python3.spec
+++ b/python3.spec
@@ -17,7 +17,7 @@ URL: https://www.python.org/
 #global prerel ...
 %global upstream_version %{general_version}%{?prerel}
 Version: %{general_version}%{?prerel:~%{prerel}}
-Release: 2%{?dist}
+Release: 2.0%{?dist}
 License: Python
 
 
-- 
2.25.1

