From c1e000fe6a102cae00877df656444bf6bdb6445d Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sun, 31 Jan 2021 13:16:42 -0800
Subject: [PATCH 10/10] Append .1.lam to Release string

---
 glibc.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glibc.spec b/glibc.spec
index 170a5bd..562bf94 100644
--- a/glibc.spec
+++ b/glibc.spec
@@ -89,7 +89,7 @@
 Summary: The GNU libc libraries
 Name: glibc
 Version: %{glibcversion}
-Release: 14.0%{?dist}
+Release: 14.1.lam%{?dist}
 
 # In general, GPLv2+ is used by programs, LGPLv2+ is used for
 # libraries.
-- 
2.31.1

