From f6e086c17f0ebf123d0378b766b2443b45f1c5f8 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sat, 22 Feb 2020 05:42:01 -0800
Subject: [PATCH 2/2] Require texinfo to build

---
 compat-libffi-31.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compat-libffi-31.spec b/compat-libffi-31.spec
index e8f75e1..c279ab7 100644
--- a/compat-libffi-31.spec
+++ b/compat-libffi-31.spec
@@ -18,6 +18,7 @@ Patch2:		libffi-aarch64-rhbz1174037.patch
 Patch3:		libffi-3.1-aarch64-fix-exec-stack.patch
 
 BuildRequires: gcc
+BuildRequires: texinfo
 %if %{without bootstrap}
 BuildRequires: gcc-c++
 BuildRequires: dejagnu
-- 
2.25.1

